docs/plan/plan-webui-phase1.md

```
@@ -1618,11 +1811,19 @@ PR 作成後、レビュー生成を待ってから `pr-review-back-workflow` Sk
 
 | テストリストID | 自動テスト名 | 追加Step | 実行コマンド |
 | -- | -- | -- | -- |
-| T27 | `ws_handler_processes_chat_events_and_send_via_chat_send` | Step 27 | `npm test --prefix web` |
-| T28 | `server_state_caches_and_invalidates` | Step 28 | `npm test --prefix web` |
+| T27 | `ws_handler_processes_chat_events_and_send_via_chat_send` | Step 27 | `npm test --prefix web` || T28 | `server_state_caches_and_invalidates` | Step 28 | `npm test --prefix web` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 9; Too many cells, extra data will be missing

(MD056, table-column-count)